nixos/modules/services/web-apps/ethercalc.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.ethercalc;
+in {
+  options = {
+    services.ethercalc = {
+      enable = mkEnableOption "ethercalc, an online collaborative spreadsheet server (persistent state will be maintained under <filename>/var/lib/ethercalc</filename>)";
+
+      package = mkOption {
+        default = pkgs.whitebophir;
```

Comment:
this matches neither defaultText nor service name?

Reply:
Oh, that was a leftover from the module I used as template. I didn't notice it in testing because I had to override the package anyway (to easily point to the package in my local nixpkgs). Will fix this in a couple moments.

Reply:
might be nice to also add a test that ethercalc comes up properly when configured through the module, if you're up for that :)